### DISCUSSION TITLE
Hybrid feature to SSG and SSR based on the date created

### DISCUSSION THREAD

**Discussion by eltonjothi** [vercel/next.js#11518] [Help] [answered]
Hello,
I haven't used the SSG on NextJS yet, but i have read about it and its really cool.
I was wondering what would be the Build time if i had to SSG for ~50k blog posts.
I assume its gonna take ~30 mins based on server configuration...

But also, i was wondering if it was possible to SSG recent 100 blog posts and SSR the remaining blog posts. ?

**Comment by timneutkens** [comment, marked as answer]
> I haven't used the SSG on NextJS yet, but i have read about it and its really cool.
> I was wondering what would be the Build time if i had to SSG for ~50k blog posts.
> I assume its gonna take ~30 mins based on server configuration...

It highly depends on the size of the page, how long data fetching takes and many other variables.

> But also, i was wondering if it was possible to SSG recent 100 blog posts and SSR the remaining blog posts. ?

Yes this is possible: https://nextjs.org/docs/basic-features/data-fetching#fallback-true

**Reply by eltonjothi** [reply]
this is awesome, thanks so much .. 🙏🏻🙏🏻
